tools/TLS Protocols Tests/TLS protocol tests.csproj

```diff
@@ -0,0 +1,30 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFrameworks>netcoreapp2.0;net451</TargetFrameworks>
```

Comment:
csproj update for linux: https://github.com/drwill-ms/azure-iot-sdk-csharp/pull/2